homework-submission/abdullahReact/main.js

```diff
@@ -0,0 +1,176 @@
+const toDoItems = [
+  {
+    id: 1,
+    description: 'Get out of bed',
+    deadLine: '2017-09-11',
+    done: true,
+  },
+  {
+    id: 2,
+    description: 'Brush teeth',
+    deadLine: '2017-09-10',
+    done: false,
+  },
+  {
+    id: 3,
+    description: 'Eat breakfast',
+    deadLine: '2017-09-09',
+    done: false,
+  },
+];
+
+const Label = ({ text, className, htmlFor }) => {
+  return (
+    <label className={className} htmlFor={htmlFor}>
+      {text}
+    </label>
+  );
+};
+
+const Input = ({ name, className, type, placeholder, defaultValue }) => {
```

Comment:
Good idea to encapsulate default html elements as your own component, in that way you can always easily add custom behaviour through-out your whole app.

Reply:
thanks.